### DISCUSSION TITLE
添加去除?p=*?t=**的功能

### DISCUSSION THREAD

**Discussion by chara26** [the1812/Bilibili-Evolved#1403] [功能建议 / Ideas]
从bilibili主站的历史打开视频，会加上?p=*?t=**的后缀，用来调到对应的集数和时间
如https://www.bilibili.com/video/av202035923?p=2&t=23  跳转到p2第87秒
建议添加一个去除这个后缀或使这个后缀失效的功能

**Comment by Geeyun-JY3** [comment]
感觉可以和 #1071，#1191 一起做

**Reply by the1812** [reply]
那个不一样的, 这个是因为 t 参数才跳时间, 而且我只能去掉参数, 影响不了行为, 该跳还是会跳的

**Reply by Geeyun-JY3** [reply]
好像也是
